picard/file.py

```diff
@@ -170,6 +171,25 @@ def save(self):
     def _save_and_rename(self, old_filename, metadata):
         """Save the metadata."""
         new_filename = old_filename
+        if config.setting["autofit_coverart"]:
+            try:
+                from PIL import Image, ImageOps
+                import io
+            except ImportError:
+                log.debug("Pillow not found. Please make sure Pillow(Python) is installed to allow\nresizing and cropping coverart images.")
+            else:
+                for image in metadata.images:
+                    resize_values = (config.setting['resize_width'], config.setting['resize_height'])
+                    image_info = imageinfo.identify(image.data)
+                    if image_info[0] > resize_values[0] or image_info[1] > resize_values[1]:
```

Comment:
Python does tuple comparison index wise, if first indices are equal, it uses the next ones to evaluate. Otherwise returns on the basis of first index comparison.